usr/sbin/revpi-bthelper

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+# referenced pi-bluetooth, bluetooth helper for Revolution Pi BT
+. /usr/bin/revpi-common
+
+set -e
+
+if [ "$#" -ne 1 ]; then
+	echo "Usage: $0 <bluetooth hci device>"
+	exit 0
+fi
+
+dev=$1
+
+# Need to bring hci up before looking at MAC as it can be all zeros during init
+# TODO: address prefix for Revpi devices needed.
+if ! /bin/hciconfig "$dev" | grep -q "BD Address: B8:27:EB"; then
+	echo Not UART-attached BT Modem on Revolution Pi
+	exit 0
+fi
+/bin/hciconfig "$dev" up
+
+# Force reinitialisation to allow extra features such as Secure Simple Pairing
+# to be enabled
+/usr/bin/bluetoothctl power off
+/usr/bin/bluetoothctl power on
```

Comment:
The rpi-bluetooth bt helper has this line. Do we need this?
```
# Route SCO packets to the HCI interface (enables HFP/HSP)
/usr/bin/hcitool -i $dev cmd 0x3f 0x1c 0x01 0x02 0x00 0x01 0x01 > /dev/null
```

Reply:
not sure, but executed ok, and function ok, I just add it